lib/kitchen/generator/init.rb

```diff
@@ -57,7 +57,12 @@ class Init < Thor::Group
       def init
         self.class.source_root(Kitchen.source_root.join("templates", "init"))
 
-        create_kitchen_yaml
+        if options[:driver].include?('kitchen-dokken')
```

Comment:
I don't particularly like this part. I imagine that if anyone is actually using this driver flag and wants more drivers defined then we are ignoring their requests.